controller/src/beerocks/bml/bml_defs.h

```diff
@@ -610,6 +613,42 @@ struct BML_NEIGHBOR_AP {
     uint32_t ap_ChannelUtilization;
 };
 
+struct BML_CLIENT_CONFIG {
+    // 1 for true, 0 for false, -1 for "not configured".
+    int stay_on_initial_radio;
+    // 1 for true, 0 for false, -1 for "not configured".
+    int stay_on_selected_device;
+    // comma-separated selected bands [2.4,5H,5L,6].
+    char selected_bands[BML_SELECTED_BANDS_LEN];
+    // Optional parameter
+    // The timelife delay of the client configuration (if configured), otherwise 0.
+    uint32_t time_life_delay_days;
+};
+
+struct BML_CLIENT {
+    // Client MAC
+    char sta_mac[BML_MAC_ADDR_LEN];
+    // Time of last client update (in Seconds)
+    uint32_t timestamp_sec;
+    // 1 for true, 0 for false, -1 for "not configured".
+    int stay_on_initial_radio;
+    // 1 for true, 0 for false, -1 for "not configured".
+    int stay_on_selected_device;
+    // comma-separated selected bands [2.4,5H,5L,6].
+    char selected_bands[BML_SELECTED_BANDS_LEN];
+    // 1 for true, 0 for false, -1 for "parameter is not configured".
+    int support_11k;
+    // 1 for true, 0 for false, -1 for "parameter is not configured".
+    int support_11v;
+    // 1 for true, 0 for false, -1 for "parameter is not configured".
+    int support_steering;
+    // 1 for true, 0 for false, -1 for "parameter is not configured".
+    int dual_band;
+    // Optional parameter
+    // The timelife delay of the client configuration (if configured), otherwise 0.
```

Comment:
you should mention what zero stands for , not sure we even want to allow the user to set lifetime zero since then there is not point adding the configuration to the DB

Reply:
@adam1985d WDYT?

Reply:
-1 as  `not-configured` seems good enough ,  just update the comment

Reply:
Updated